static/sass/styles.scss

```diff
@@ -34,12 +38,11 @@ $p-small-lh-diff: map-get($line-heights, default-text) -
 @include vf-p-icon-user-group;
 @include vf-p-icon-video-play;
 @include vf-p-icon-warning-grey;
-@include vf-p-icon-begin-downloading;
-@include blog-p-card;
+//@include blog-p-card;
 @include canonical-u-animations;
 @include canonical-p-stepped-list;
 @include canonical-p-buttons;
-@include canonical-p-card;
+// @include canonical-p-card;
```

Comment:
Same here

Reply:
uncommented, same as above, to be removed at the end of the project